audio/src/main/java/edu/artic/audio/AudioDetailsFragment.kt

```diff
@@ -26,6 +26,7 @@ import edu.artic.db.models.AudioFileModel
 import edu.artic.image.listenerAnimateSharedTransaction
 import edu.artic.media.audio.AudioPlayerService
 import edu.artic.media.refreshPlayBackState
+import edu.artic.ui.util.isNotEmpty
```

Comment:
Because the observable streams in `AudioDetailsFragment` do not specify types, the existing `String.isNotEmpty` calls are converted directly to `Uri.isNotEmpty`. This is thus the only line change needed to make it work.